### PR TITLE
Update readme frontend

### PR DESCRIPTION
**Pull request by elBeato**
update according guidelines: working document, please feel free to add or change the content!

https://github.com/sopra-fs22-group-36/screw-your-neighbor-react/blob/feature_ReadMe/README.md

## Issue: 
#130

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,17 +56,45 @@ Then run again `npm run openapi-generate`
...
+### Game playing
+
+### Winner :tada: / Looser :screwdriver::facepalm: page 
+
+## Roadmap
+### Top features to contribute
+- Clear visualisation who's turn is right now
+- Create your own avatar 
+- Choose backside image for cards
+- Change frontend to socket.io solution
```
Comment (salwil) [inline README.md:90]:
```suggestion
- Sound effects when playing card, winning / loosing trick, winning the game etc.
```

File: README.md [hunk 2]
```diff
@@ -35,17 +56,45 @@ Then run again `npm run openapi-generate`
...
+
+### Winner :tada: / Looser :screwdriver::facepalm: page 
+
+## Roadmap
+### Top features to contribute
+- Clear visualisation who's turn is right now
+- Create your own avatar 
+- Choose backside image for cards
+- Change frontend to socket.io solution 
+
```
Comment (salwil) [inline README.md:91]:
```suggestion
- Sound effects when playing card, winning / loosing trick, winning the game etc.
```

File: README.md [hunk 3]
```diff
@@ -35,17 +56,45 @@ Then run again `npm run openapi-generate`
...
 in [Lobby](src/components/views/lobby/Lobby.tsx).
 
+### Jitsi API
+We implemented the Jitsi API into our game to give the players an option to video chat and 
+to fulfill the requirements given to us by the SoPra Team.
+
+
+## Illustrations
+
+### Lobby & Lobby
```
Comment (salwil) [inline README.md:77]: Lobby & Lobby?

Comment (elBeato) [reply README.md]: ubs...:-D

**Review by BacLuc — APPROVED** [review]

**Review by CarmenUZH — APPROVED** [review]

**Pull request merged**